### PR TITLE
PSP-3889:Field label suggestions for Property Information tab

### PR DESCRIPTION
**Pull request by Maheshbabu28**

### CODE REVIEW

**Review by devinleighsmith — COMMENTED** [review]
I have a question out to Dev on this one, will approve when I receive a response.

**Comment by Maheshbabu28** [comment]
image

image

**Review by devinleighsmith — CHANGES_REQUESTED** [review]
see correction from Dev

File: frontend/src/features/mapSideBar/tabs/propertyDetails/detail/PropertyDetailsTabView.tsx [hunk 1]
```diff
@@ -137,7 +137,7 @@ export const PropertyDetailsTabView: React.FunctionComponent<IPropertyDetailsTab
             </SectionField>
           )}
           {isAdjacentLand && (
-            <SectionField label="Adjacent land">
+            <SectionField label="Adjacent Land Type">
```
[the remark this answers is not in the document]
Comment (Maheshbabu28) [reply frontend/src/features/mapSideBar/tabs/propertyDetails/detail/PropertyDetailsTabView.tsx]: image

**Review by devinleighsmith — APPROVED** [review]

**Pull request merged**